app/resources/css-default.template

```diff
@@ -605,6 +607,16 @@ td.submit {
   border: 2px solid #e6e6e6;
 }
 
+.query.seek .field {
+  display: block;
+  margin: 8px 20px;
+}
+
+.query.seek .label {
```

Comment:
Should I add a comment for this too? 
/* Edit label in seek query because label is too long */ ?